api/proto/teleport/legacy/types/types.proto

```diff
@@ -6612,6 +6612,27 @@ message SMTPSpec {
   string host = 1;
   // Port specifies the SMTP service port number.
   int32 port = 2;
+  // StartTLSPolicy specifies the SMTP start TLS policy used to send emails over
+  // SMTP.
+  SMTPStartTLSPolicy start_tls_policy = 3;
+}
+
+// SMTPStartTLSPolicy defines the start TLS policy used to communicate with the
+// SMTP service.
+enum SMTPStartTLSPolicy {
+  // SMTP_MANDATORY_START_TLS means that SMTP transactions must be encrypted.
+  // SMTP transactions are aborted unless STARTTLS is supported by the
+  // SMTP server. Recommended for all modern SMTP servers.
+  SMTP_MANDATORY_START_TLS = 0;
+
+  // SMTP_OPPORTUNISTIC_START_TLS means that SMTP transactions are encrypted if
+  // STARTTLS is supported by the SMTP server. Otherwise, messages are
+  // sent in the clear.
+  SMTP_OPPORTUNISTIC_START_TLS = 1;
+
+  // SMTP_NO_START_TLS means encryption is disabled and messages are sent in the
+  // clear.
+  SMTP_NO_START_TLS = 2;
```

Comment:
Usually we leave the 0 enum case for the UNSPECIFIED case.

Also enum cases must start with the enum name I think, else the buf linter will not let your PR pass